android/src/app/organicmaps/widget/placepage/PlacePageView.java

```diff
@@ -194,6 +193,8 @@ public class PlacePageView extends NestedScrollViewClickFixed
   private CountryItem mCurrentCountry;
   private boolean mScrollable = true;
 
+  private double mLastKnownNorthAzimuth = 0;
```

Comment:
Why do you think it is needed? What is the benefit of showing some potentially (very) outdated azimuth? Wouldn't it confuse users, for example, when POI was selected, and location updates were _disabled_ in the system settings?

Reply:
The azimuth is only displayed if the location is available, so this case should no happen.

Reply:
What if user disables location in the system while using OM with an open place page?

Reply:
The arrow and distance is still shown, but if the user closes and reopens the place page, they are properly hidden. On current master the behavior is the same but only the distance is shown, which is still wrong.

IMO even if this is not the best way to handle this (but it would need a bigger refactoring), this is still better than what we currently have in master as only showing the distance without the arrow feels like a bug.

Reply:
There likely was good reasoning behind it: without a gyroscope, you don't know the direction. But in most cases, you can't move too fast to make the distance outdated. So it makes sense even if the location was _just recently_ disabled. Of course, if the location is _too_ outdated, then the distance should not be displayed.

Reply:
I can try to disable both distance and direction when the location is disabled/lost, instead of keeping the old behavior, but it might be difficult.

Reply:
My point was to remove direction, but leave the distance, if there is no fresh GPS data.